### PR TITLE
feat:add second tech contact

### PR DESCRIPTION
**Pull request by kurokn**
- add the second technical contact attribute;
- for current permission level, the sec_tech_contact can be changed by any admin/po/technic owner account;

### CODE REVIEW

File: app/pages/api/realms/one.ts [hunk 1]
```diff
@@ -172,9 +184,22 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
             division=$7,
             branch=$8,
             updated_at=now()
+            second_technical_contact_email=$11,
```
Comment (junmin-bcgov) [inline app/pages/api/realms/one.ts:187]: can you double-check this indexes are the correct ones?

File: app/types/realm-profile.ts [hunk 2]
```diff
@@ -13,6 +13,9 @@ export interface RealmProfile {
   technical_contact_email: string;
   technical_contact_idir_userid: string;
   technical_contact_name: string;
+  second_technical_contact_email: string;
+  second_technical_contact_idir_userid: string;
+  second_technical_contact_name: string;
```
Comment (junmin-bcgov) [inline app/types/realm-profile.ts:18]: actually we don't need `second_technical_contact_name` as it is dynamically added via Keycloak API calls, can you double-check how `technical_contact_name` has been populated at backend?

**Review by junmin-bcgov — COMMENTED** [review]
in overall, looks good!

File: app/pages/api/realms/one.ts (hunk 1 above)
Comment (tzhang200) [reply app/pages/api/realms/one.ts:187]: Likely not, they are $15 and $16 at line #119

**Pull request closed without merging**